apps/trading/client-pages/markets/markets-page.tsx

```diff
@@ -125,78 +166,355 @@ export const MarketsPage = () => {
+                    'text-muted': !isMarketTypeSelected(marketType),
+                  }
+                )}
+                id={marketType}
+                onClick={() => {
+                  let types = marketTypes;
+                  if (isMarketTypeSelected(marketType)) {
+                    types = [];
+                  } else {
+                    types = [marketType];
+                  }
+                  setMarketTypes(types);
+                }}
+              >
+                {marketTypeFilterOptions[marketType]}
+              </button>
+            );
+          })}
+        </div>
+
+        <div className="flex justify-end gap-2">
+          <div>
+            <MultiSelect placeholder="Status" trigger={marketStateTrigger}>
+              {Object.keys(marketStateFilterOptions).map((key) => {
+                const marketState = key as MarketState;
+                const isChecked = marketStates.includes(marketState);
+                return (
+                  <MultiSelectOption
+                    checked={isChecked}
+                    onCheckedChange={(checked) => {
+                      if (checked) {
+                        setMarketStates(uniq([...marketStates, marketState]));
+                      } else {
+                        setMarketStates(
+                          marketStates.filter((s) => s !== marketState)
+                        );
+                      }
+                    }}
+                    key={key}
+                  >
+                    {marketStateFilterOptions[marketState]}
+                  </MultiSelectOption>
+                );
+              })}
+            </MultiSelect>
+          </div>
+          <div>
+            <MultiSelect placeholder="Assets" trigger={assetTrigger}>
+              {assetFilterOptions.map((asset) => {
+                const chainName = getChainName(
+                  asset.source.__typename === 'ERC20'
+                    ? Number(asset.source.chainId)
+                    : undefined
+                );
+                const isChecked = assets.includes(asset.id);
+                return (
+                  <MultiSelectOption
+                    checked={isChecked}
+                    onCheckedChange={(checked) => {
+                      if (checked) {
+                        setAssets(uniq([...assets, asset.id]));
+                      } else {
+                        setAssets(assets.filter((a) => a !== asset.id));
+                      }
+                    }}
+                    key={asset.id}
+                  >
+                    <span>{asset.symbol}</span>{' '}
+                    <span className="text-xs">({chainName})</span>
+                  </MultiSelectOption>
+                );
+              })}
+            </MultiSelect>
+          </div>
+          {/** MARKET NAME FILTER */}
+          <div className="w-2/3">
+            <TradingInput
+              prependIconName={VegaIconNames.SEARCH}
```

Comment:
This icon looks different (chunkier) to the one in the market selector dropdown, which is the correct one